### ISSUE TITLE
Role datums feature request: Force lings to change identities either compulsively or routinely

### ISSUE THREAD

**Issue by kilozombie** [d3athrow/vgstation13#19774]
I would actually give this a go on the current build but since role datums are slowly encroaching like stealth cult on lowpop, I'm making it a feature request for the folks with more knowledge of that code.

Changelings should no longer be able to swap identities freely, including through UI injectors. Instead, they should instantly and easily adopt the identity of anyone they absorb. Here's why.

_1. Why does Changeling need a change?_

**Current Changeling is played in a boring way.** Changelings use their abilities and outside tools in order to capture people and absorb them in hidden locations. For these players, they're taken out of the round with very little input on the matter. For everyone else, it's the equivalent of people going braindead in dorms or going vault hunting or just fucking off from the department. There's nothing interesting about people slowly disappearing, because there are far more innocuous reasons people disappear nowadays than terrifying reasons. 

**Changeling doesn't generate paranoia, it generates slow, completely uninteresting murderbone from one person.** It's like playing any of the other gamemodes in which you end up in the wrong place at the wrong time and get taken out of the round forever while everyone else forgets about you. There is nothing wrong with being taken out of the round in spess-- but there's something inherently very shitty about your death meaning _nothing_ except getting the changeling one step closer to deathsting.

**A discovered changeling is like finding any other criminal.** Like any other criminal who has murdered a lot of people, they generally have their name attached to every single kill. It's always been that one person murdering all those folks. If you know somebody is a changeling but the changeling doesn't know that you know, you will forever be aware of this fact, because changelings still have practically no incentive to change their form. There is no betrayal in finding out that the shady Assistant who's been lurking around Maintenance all round was a murderer, and there is no surprise in finding out it wasn't just any murderer, but a murderer with meme stun tools and the ability to go horror form (see: "b-b-but changeling can go loud!" bullshit fodder).

_2. Why would this change have a positive impact?_

**Forcing you to change identity means you have to play the part.** You can no longer go back to your department and act like nothing's changed, which is what you do as any other antagonist with any other jectie to murder people. Instead, either you don a gas mask and become anonymous (which makes you instantly more likely to get in trouble with just about everyone), or you take on the identity of the person you just killed. Now, if you're discovered anytime after your first murder, it will be at _least_ a revelation for the person who found you out. Finding out your coworker has been somebody else- for God knows how long- radically changes the context of every other one of your interactions.

**Forcing you to play the part means you're better off researching and learning.** While you can forego seeming genuine in your interactions with your newfound coworkers, this now opens you up to a new route of being discovered: by the people who knew that person better. While questions like "what did i say to you at the start of the shift?" shouldn't be metagamed for an easy valid (make "I forget" an always-valid option), everything along those lines builds suspicion and paranoia even among friends who aren't changelings. If somebody finds a husk in Maintenance or some discarded clothes near your new department, there's immediately going to be questions. So you'd better learn who you're murdering and how they play, so it's that much more surprising and interesting when the secret gets out, whether upon being found out or as the round ends. You're encouraged not just to roleplay a little, but try and actually get to know your future victims and pay more attention in the round for your own benefit.

**Forcing you to play the part means the victim gets to watch you impersonate them.** This is sort of an oft-forgotten part of changeling that I've only experienced 1-2 times in the rare instances someone actually steals my identity-- I'm no longer just watching some other guy dunk six other hapless victims in Maintenance, I'm watching somebody pretending to be _me_. And if they head to my department and start chatting up with my coworkers, the whole time I'm either cursing the fact that they haven't noticed or cheering them on when they realize it's a fake. This inherently makes the round more engaging, not just for the people that are looking for the changeling, but also its many victims.

_3. What could go wrong?_

**It's possible that people won't play the part and pretend to be their victim**, but I think that it's a server culture thing, at some level. With time, once people know that a coworker could, at any time, be a changeling- _and that a husk means somebody's being impersonated_- people may start paying more attention to their coworkers and pals. It _will_ become more routine to have some kind of codeword, and changelings may need some kind of other tool to adapt to that change. I do think that this issue will sort itself out in time, though.

An extremely radical idea to help that issue would be to implement something similar to Bay's flavor-only randomly pairing relationship system. To summarize: you can set it up so that at roundstart, you'll be randomly assigned a friend/enemy with somebody who also enabled it in their options, along with some flavor text about why that might be. It doesn't affect rounds, but it does give you knowledge about a given person, and with a wide enough connective net between players, everybody would know _something_ about someone else that a changeling might not have an answer for, fueling more suspicion.

**It's possible that people will metagame and decide that not knowing a codeword or tiny detail means it's a free valid.** But this is sort of a moot point: ban people who metagame, and make it always fair game to answer "I don't remember" or "Why do you want to know?" I know for a fact I'd start replying with that as a nonantag so that it remains consistent with when I'm not.

**It's also possible that the differences between mannerisms won't be great enough on our current level of roleplay to facilitate this "playing the part" business.** While some people play static names and even random names in a way that's consistent (the former between rounds, the latter in a given round), other people sort of just play however fits the part at a given time-- and that's completely fine! On the other hand, there are still nonverbal cues like movement or response time that could indicate at least some level of suspicion.

One thing is if people have to rely more on suspicion, having some way to up the ante on suspicion would be valuable. We've been opposed to genuine changeling tests in the past, but if it were arduous enough and nonviable for mass-testing, it might actually fit well in this new setting. If your buddy has been acting strange for a long time, you can bring them in for a test.

**It's possible that forcing changelings into this style is too far, and in that case, here's a great alternative** An anon in the thread brought up the idea of genetic deformation-- using your powers will deteriorate your genetic makeup until you're clearly visible as a changeling, forcing you to switch to a new identity. I think it's got a lot of potential, but changelings don't _inherently_ need to use their powers-- there are other ways of murdering people secretly that completely forego powers which already exist in the game, and incentivizing changelings _not_ to use their only differentiating feature seems massively counterproductive. 

One idea to counter that would be to make genetic deterioration time-gated instead, giving you 20 minutes or half an hour to find new DNA. Or, perhaps, it could be a mix-- you have 45 minutes to find a new body, but using abilities shortens that timer. As the timer runs out, you become more visible-- @Blithering pointed out a good system way down below.

**It's possible that knowing the round isn't ling would make you stop searching for changelings in your department on a meta level**... which is exactly why this feature sort of necessitates role datums. So that's about that.

**Comment by SonixApache** [comment]
- Forcing lings to play the way **you** want to play isn't going to change the way ***they*** want to play apart from maybe unchecking ling.
- If people didn't bother trying to actually impersonate people before, they're not going to do it now, even with your 'incentive' -- rather, once you succ someone, you'll just go fast into wordlessly succing other people that might be near you
- It will be metagay'd. If you ban for that level of metagay, we'll lose another decent chunk of the population.
- It's not going to make people more suspicious of eachother -- either you fail the metacheck or you don't. And if you do, (eg. "I don't remember"), you're getting a ticket straight to the cremator.

tl;dr this is dumb, play antag the way you want and pray for better players

**Comment by kilozombie** [comment]
> It will be metagay'd. If you ban for that level of metagay, we'll lose another decent chunk of the population.
> It's not going to make people more suspicious of eachother -- either you fail the metacheck or you don't. And if you do, (eg. "I don't remember"), you're getting a ticket straight to the cremator.

If you read my post, you'd be aware that it's a very simple rule change away from being perfectly fine. It's extremely valid roleplay to say "I don't remember" and it's also perfectly valid to get suspicious of people who do this, but we already ban people who try to collect valids on stuff this baseless, so why would we stop now? The level of metagay we're banning for is "murdering on basically no evidence", which isn't exactly foreign.

> Forcing lings to play the way you want to play isn't going to change the way they want to play apart from maybe unchecking ling.

I don't think that spending a 2 and 1/2 hour round deathstinging people is all that engaging for the people who do play Changeling. Sonix, you already lock off playstyles that people enjoy regularly with your "suggestions", don't get on a high horse and act like 1. people actually _enjoy_ playing Changeling this way and 2. that making a change to a gamemode is somehow the most evil act.

Forcing changelings to play with a new restriction will not force them to play in a certain way, it will force them to actually think through their actions. It has always been, and will continue to be, perfectly valid to just sulk in Maintenance waiting for prey. This is purely to discourage playing Changeling similarly to any other antag that walks away from their department, murders someone secretly, and walks right back into their department like nothing happened. We have several antags who already do that. Changeling has the potential to be so much more.

> If people didn't bother trying to actually impersonate people before, they're not going to do it now, even with your 'incentive' -- rather, once you succ someone, you'll just go fast into wordlessly succing other people that might be near you

This is sort of nonsensical and reads like a meth head wrote it, but I'm going to try and deconstruct it.

> If people didn't bother trying to actually impersonate people before, they're not going to do it now,

This change proposes to take away one of their alternatives (walking back to their department). If impersonating people is relatively easy in many cases (which I would say it is), this will encourage people to get better at noticing people with changed behaviors, and encourage changelings to get better at impersonating people. It's a feedback loop arms race that makes both sides a more enjoyable experience than just "that person is probably the powergaming ling who murdered like 8 people" and "i'm gonna go murder 8 people and do my job in the meantime"

> even with your 'incentive' -

My post has no incentives. Though changelings get a free identity swap, this is a bigger nerf than buff to Changeling. The incentive is "play better and you will get better results", which isn't exactly new.

> rather, once you succ someone, you'll just go fast into wordlessly succing other people that might be near you

I lose the plot on this part. Like I said, you'll always be allowed to go wordlessly succing people in Maintenance, but that's absolutely the easiest thing to avoid as a victim (just don't approach fucking crazy masked people in Maintenance). But if you don't do that, this actually offers you a very quick way to get off scot free-- succ someone, swap your clothes with theirs, and leave everyone in the dust as to who actually got husked, even moreso if you take their head with you.

Why would you instantly go to wordlessly succ other people that "might be around you"? What does that mean? Is this another Sonix-ism that escapes true understanding? If so, that's very artful, but it doesn't particularly belong on Github.

Let me reiterate that one of the prominent objectives of this change would be to make Changeling a unique role that differentiates itself from the other roles where you're encouraged to secretly murder people over and over. Why is it so incredibly foreign to make one of your abilities mandatory? Why is it so incredibly foreign for changelings to have an obstacle they have to work around that's unique from other antagonists?

> play antag the way you want and pray for better players

This isn't an issue of "play antag the way you want". The most efficient way to play Changeling right now is also the most boring way (for yourself and others). This isn't players being bad, this is a design issue. **Just like jecties.**

**Comment by TurboGay** [comment]
I'd agree with a change like this if it didn't mean having to swap out of all of your equipment/clothing on each kill, because realistically that's what you have to do in order to maintain an identity and being forced to do it every time you extract DNA would just feel tedious to do.

**Comment by kilozombie** [comment]
@TurboGay Would you be in favor of some kind of changeling tool that allows you to quickly swap equipment? I'm not sure exactly how to keep it themed correctly with the role, but it'd facilitate a lot of cool situations.

**Comment by Caboozles** [comment]
I'm fine with this on the sheer premise that Ling is a beyond unfun gamemode for all involved, except for that ling.
People going away over a three hour round only for a horror form to pop out is just a "That explains why fucking nothing was happening" sort of feel.

**Comment by TurboGay** [comment]
@kilozombie I think it'd be an interesting tool and could even work in the uplink for something like the geneticist, which also badly needs something that closer fits their job instead of a gun. I'd be in favor of it for sure.

**Comment by Kammerjunk** [comment]
I don't really care about this either way right now but I remember a round where someone succed and became Dummo, if you remember him, and just walked around acting like a normal guy with nobody calling it out

**Comment by kilozombie** [comment]
@Kammerjunk It's a really good point that currently you can take most people's identity with almost no callout because nearly every existing changeling will just keep their current identity. But I do genuinely believe that server culture would shift to pay more attention to mannerisms, and _fast,_ if it was a more common occurrence (and an indicator of danger).

**Comment by iFlashYou** [comment]
Indirect buff to anyone that starts with an implant. 
And nobody will start to go around and try to play ling how you want to, they will either turn it off or play it as tator without uplink. It just takes to long to succ someone, change their outfit with yours and then to dispose the body, while you have to maintain your stealth in all of this.

**Comment by kilozombie** [comment]
@iFlashYou It's not a stretch to give changeling some way of swapping outfits quickly, and considering their primary ability is _literally morphing flesh_ swapping implants also seems perfectly reasonable. These aren't roadblocks so much as they are things that would obviously have to be changed with a balance shift like this.

**Comment by SonixApache** [comment]
What if instead of "succstended, except it changes directions sometimes" ling was changed to actually have something to do for the poor bastards getting succed?
Or what if lings were harder to kill, rather than just "stun, lase, decap" ?

**Comment by kilozombie** [comment]
Please offer even a single suggestion for giving things to do to the poor bastards getting succed.
For instance: changelings being able to silently copy somebody's body and implants while they're still alive to play the "which one is the real Greg?" game in some way?

Please offer even a single suggestion for how to make lings harder to kill.
For instance: headcrawlers like in The Thing, which is the main inspiration for current changeling?

I already mentioned in the OP that watching somebody impersonate you is already a "thing to do", it's engaging especially if people are aware of the feature.

**Comment by SonixApache** [comment]
Players that get absorbed become part of the ling and can talk to it. The ling can mute them.
The ling can remove parts of itself. Every part of the ling is the ling. The ling isn't fully destroyed till all parts are gone. Whenever the main body of the ling dies, it can transfer itself to any of the parts to do stuff with and slowly regrow into a full ling.
The absorbed players can be transferred to the parts. These parts can move and do stuff, but can't grow into a full ling.

Or you could just give lings ling icons, like every other 'team' antag gamemode. Might change things up a bit. Add a ling-sting that turns people into lings, maybe. Lesser lings?

Again, you've written a few paragraphs about why people should be forced to play the game the way you want them to play, and again, it's pointless. Giving them more tools to work with, and/or making it easier for them to give the rest of the players a fulfilling / 'fun' round would be a better approach.

**Comment by kilozombie** [comment]
> Again, you've written a few paragraphs about why people should be forced to play the game the way you want them to play, and again, it's pointless.

Writing about yourself again, eh?

No, this isn't about me trying to force players to play the game the way I want them to play, and massively distorting my point to serve your own wrong narrative is a wacky tactic. I explain in the OP why there's a problem with how Changeling is currently constructed and why instituting a _roadblock_ along with a _buff_ would force the _mode as a whole_ to be better. You cannot say, in any way, that Changeling is currently played in an interesting way, when most of the time it's somebody just dragging people to maint for kills again and again and again. Once again, this is so similar already to existing gamemodes while also being boring for everyone involved that a change is not only necessary, it's baffling why one hasn't been made already by _someone_.

You clearly have an interest in changing the mode. Even your proposed change would force changelings to play in a new way-- isn't that inherent in literally any change to a gamemode? If traitors no longer had an uplink, they would be forced to play in a new way. If changelings were made into limb Katamari, they would be forced to play in a new way. If changelings could no longer keep their identity after absorbing someone, they would be forced to play in a new way. That isn't some selfish endeavor to get them to play in _my_ way-- it's not like this idea is some kind of ego project. This issue is highlighting an idea on how to improve the gamemode by making an ultimately major, but very simple, change.

I am confused as to why changing the flow of the game is such a boogeyman for you, while in the very same post you propose something that would require tons of coding and would change how every single changeling plays the game. It's almost like you don't actually have a legitimate objection to the idea I put forth in the OP and you're just searching for reasons to validate your initial distaste.

**Comment by kilozombie** [comment]
> The absorbed players can be transferred to the parts. These parts can move and do stuff, but can't grow into a full ling.

Oh boy, I sure love "getting killed by the antag lets you BECOME the antag"-lite, that definitely won't lead to anything weird ingame.

> Or you could just give lings ling icons, like every other 'team' antag gamemode. Might change things up a bit. Add a ling-sting that turns people into lings, maybe. Lesser lings?

This is better and would facilitate some of the "growing hivemind" stuff that is classic to the bodysnatcher/Thing/changeling mythos. Unfortunately, it does lose the "this isn't the man I once knew!" factor if it _is_ actually your coworker, just now he's a changeling. Then it just becomes Revs. Remember how we're trying to avoid being like the other gamemodes?

Taking somebody's identity is even more crucial and core to the bodysnatcher/Thing/changeling mythos than the growing hivemind stuff, but if you can consolidate both of them, that'd be two great changes to benefit an awful gamemode.

**Comment by SonixApache** [comment]
>Writing about yourself again, eh?

You can leave sentences and remarks like these out, it doesn't really help the discussion in any way.

>I am confused as to why changing the flow of the game is such a boogeyman for you
>It's almost like you don't actually have a legitimate objection to the idea I put forth in the OP

Because I don't see your idea as something that will actually change people's playstyles apart from causing a slight change in which department gets the next succ.

**Comment by kilozombie** [comment]
>Because I don't see your idea as something that will actually change people's playstyles apart from causing a slight change in which department gets the next succ.

Alright, then I'll do my best to convince you by using the highlights of my first post.

>Forcing you to change identity means you have to play the part. You can no longer go back to your department and act like nothing's changed, which is what you do as any other antagonist with any other jectie to murder people.

This is objective. You _cannot_ act normal after you've done a big boy succ-- you need to be willing to improvise. You now have a facade that you must keep up. This doesn't just change which department gets the next succ-- it makes you have to convince the next department that _you belong_. As server culture shifts, this is, in my mind, not going to be easy, and going to require actual improvisation and smarts.

>Forcing you to play the part means the victim gets to watch you impersonate them.

Even if you are absolutely right, even if things stay similar to how they currently are... does this make the change a bad thing, or just a mild one? This gives something actually interesting for victims to watch and it still gives a new route for unwitting players to try and identify threats. Now they know that if a coworker starts acting funny, they might want to be careful. What is the downside of this change? Well, let's address some that you might bring up.

>It's possible that people won't play the part and pretend to be their victim.

It's possible that nobody will initially pay attention to coworkers acting funny. But after the first one or two times they're succed by a coworker acting funny, people will pay attention, and changelings will have to try harder to compensate. Both sides now have an active goal to work towards besides "when you succ someone make sure nobody finds you" and "literally anyone could be changeling at any time"

>It's possible that forcing changelings into this style is too far, and in that case, there are alternatives. 

There are other ways to encourage identity theft. Whether it's a carrot or stick model, either one still fulfills the goals that I've outlined in this entire issue, of making changeling more interesting.

It seems to me like you are unwilling to even consider this as an idea, and this still seems to me like a bias. Maybe I'm projecting a little, but it's easy to have initial opposition to an idea on principle and try to find a reason not to like it. But you saw what people were trying to say with the jecties debate, so I'm curious still about what you really don't like about this idea. Just the whole of it? The way I phrased it?

**Comment by SonixApache** [comment]
What I don't like is taking away the opportunity to do what the player wants to do. Want to be the captain to shitpost on comms? Too bad, kill and absorb him (and avoid the SPS memes) first. Want to argue with 'yourself' over the radio? No chance. Want to switch identities because someone figured you out? No-go, unless you spend another minute or two murdering and standing still to succ someone. Not being able to switch identities on the go seems like a major detriment to the antag and mode as a whole. And that aside, I think it'll be grossly *ineffective*, since the entire mechanic can be foiled with a gas mask and an itchy trigger finger.

That all said, I'm not necessarily *against it*. Code it, give it a run, see if anything changes for the better, maybe I'm retarded. I just think it's a waste of time in terms of effectiveness, and a nerf in terms of opportunities.

**Comment by kilozombie** [comment]
Thanks, those are some real concerns that do deserve addressing. I will reiterate _again_ that being able to use a gas mask and an itchy trigger finger has always been an option and this would keep it that way intentionally, but it's an options-limiter and Security on Code Blue loves nothing more than randomly searching your suspicious ass.

Again, one alternative option was to incentivize you to change bodies regularly (but still allow you to have several at a time) would be to have each of the bodies have a limited lifespan (that only ticks down when being used)-- say, 30 minutes before it's clearly visible as a changeling, and 45 minutes until it's unusable at all, or something along those lines. That encourages you to body snatch and do some kind of changing, while also facilitating you to do it in any way you like. This, _again_, I mentioned in the last post I made as well as the OP.

**Comment by Kammerjunk** [comment]
can a mechanic simultaneously force antags to play a certain way and be grossly ineffective at forcing antags to play a certain way

**Comment by Spessgontran** [comment]
>It's like playing any of the other gamemodes in which you end up in the wrong place at the wrong time and get taken out of the round forever while everyone else forgets about you. There is nothing wrong with being taken out of the round in spess-- but there's something inherently very shitty about your death meaning nothing except getting the changeling one step closer to deathsting.

All i need to read in order to know this PR is bullshit. There are various ways you can come back into the game, may it be deleric mommi/pAI/mouse/golem/Diona/posibrain/adminbus (try it, ask to be a corgi or something else next time), and you chose to shit on people that likes playing as X antag instead? It sounds like you refuse to be anything else than your special retarded snowflake of a character and absolutely want to see if someone else can nail your shitty personality for your own fun if you ever get snatched up.

Here, read the second serbia rule:

" You're playing a multiplayer game. Sometimes you're having fun with everybody. Sometimes other people will be having fun at your expense. Sometimes you'll be having fun at their expense. Not every round will be flowers and sunshine. Antagonists are there to ensure that. However, if you think you're being griefed, make sure to adminhelp it."

Changeling is fine as it is, especially now that Datums are here. If you want to actually do your part, make Vampire better. Or do yourself a favor and code something that will bring you back to the round instead of trying to make something that is already fine the way it is worse.

**Comment by kilozombie** [comment]
>Changeling is fine as it is, especially now that Datums are here. If you want to actually do your part, make Vampire better.

All I need to read in order to know this post is bullshit. :)

Ask people who play on the server and you'll notice a trend of complete distaste for the gamemode. Even folks who like the gamemode agree almost universally that it should be made better _somehow_.

Moreover, the point of that paragraph isn't for me to complain about dying or even dying meaninglessly-- but that that's _how the other solo antag modes already operate_. There's absolutely nothing functionally unique about changeling and every feature of changeling serves to create boring rounds. Datums help in that now changelings don't have to be the only antagonist, but they're still overwhelmingly in need of an identity as a gamemode.

I outline the exact reasons it needs a change. There are multiple. I outline the reasons that this suggestion could help all of those problems. There are multiple.

 You are shitposting, straight up. Please go do it in one of the designated shitposting spaces like the thread.

**Comment by SonixApache** [comment]
I mean, as abrasive as you can be in your reply, you're not really seeing the point that forcing feature loss won't actually address the issue that lings as solo antags act the same as other solo antags, which is (stealth) murderbone.

You also claim that `every feature of changeling serves to create boring rounds`, but in a thread where you advocate to *remove* ling features in favor of forcing them to use one of their features.

**Comment by Spessgontran** [comment]
Not everyone plays the same way. When i get changeling, i tend to become my victim or steal the appearance of someone. I also go out of my way to fuck with people with their voices.

If you start forcing people to play in a certain way for this antagonist, you'd have to remove more than half the fun stuff they have like change voice, steal sting, and various ability that promote fun.

It's like trying to change medbay when you don't play it anymore but find it too easy. The real losers are the people who don't play it or want to learn it.

Also, you'd be a perfect fit for /tg- er, TG. When was the last time you played?

**Comment by kilozombie** [comment]
@SonixApache This actually _does_ address the issue that lings act the same as other solo antags, by making the problem of murder in any context more complicated. Just making baseless claims about the thing I'm advocating gets you nowhere.

**Forcing you to change identity means you have to play the part.** You can no longer go back to your department and act like nothing's changed, which is what you do as any other antagonist with any other jectie to murder people.

Sure, it's stealthy, or maybe it's loud with identity changes in-between! If we go with the alternate proposal (each body has a limited lifespan, so change now and then), you have an even larger plethora of possible options, while still needing to use _some_ part of what makes Changeling interesting.

Also, it's completely nonsensical to say "Well, if Changeling's current features make it bad, removing one won't do anything!" These features are all set in the context of the other features they synthesize with. Right now, Changeling's ability to change appearance and voice and such is hampered by the rest of the context-- that you're still best off just playing it cool and not hopping between identities. I think it's wonderful that people do that! But if we're talking about the _server_'s health, here, which I very much am, it's unhealthy that the encouragement is towards a boring playstyle.

Like jecties. The comparison goes pretty deep there.

This suggestion has never implied that Changeling would just lose features and gain nothing, instead-- I think that obviously with a shift this big there would need to be more tools and some tools would be need to be made more viable. Initially I thought these should be minor, but now I think they should be major. Being able to copy someone's appearance then steal their outfit quickly would be incredibly helpful for all sorts of situations, opening you up to fast, tricksy gameplay.

You are much more opposed to the idea of change than the intent or implementation, it feels like.

@Spessgontran It does feel like you read Sonix's replies about my proposal much more than you read the proposal. This isn't a matter of something people find fun being torn to shreds because of some kind of ego project of mine- which is definitely something we see on Github here- this is my genuine attempt to try and patch up the gamemode for the better. I never said this was perfect and that it wouldn't need changes in order to slot into normal gameplay-- in fact, I've said the opposite several times.

You, as someone who plays changeling in a way that actually differentiates it from other solo antags, have valuable experience. What other ways can we find cool and completely unique encounters using changeling's theme as a shifting, amorphous being?

Because that is the _intent_ here. You can point at a bogeyman (buddy, I've played exclusively /vg/ for two years) and you can even parrot things Sonix already said that remain invalid points, but this suggestion isn't here to remove things that people use for fun. It's to address the glaring problem that most people in a round don't find changelings fun, except for _maybe_ the changeling. There is a big ol' reason that lingstended is a thing.

**Comment by strawberrytan** [comment]
last time I impersonated someone I got the good old account info check done on me

**Comment by kilozombie** [comment]
> last time I impersonated someone I got the good old account info check done on me

Make notes transfer to a changeling when they succ someone.

**Comment by Blithering** [comment]
I got a word doc typed up for an ideasguy ling rework, but the basic gist for disguise would be:
Your current identity has a genetic integrity meter, which degrades the more abilities are used. As your genetic integrity drops, it becomes easier to identify the ling as not being human. (~90% autopsy will have a note about weird DNA, ~60% full body scan will wig out and claim the body has no organs, ~40%  health scanners will read -100 health at all times, 20% or less the person appears to be rotting) The only way to regain genetic integrity is to transform, which resets the meter and deletes your old identity, so no swapping back and forth.

**Comment by kilozombie** [comment]
@Blithering That sounds fantastic. But like I mentioned in the OP, I think that it should be timer-based, or at least have some timer going off constantly. Like-- it _very slowly_ degrades your body, and abilities speed it up. Else there's the obvious workaround of playing Changeling ability-less.
